src/api/Feed.js

```diff
@@ -95,6 +103,75 @@ const getItemWithPendingReply = <T: { replies?: Array<Comment> }>(item: T, reply
     return { replies: [...replies, reply], ...rest };
 };
 
+const parseFileActivitiesResponseForFeed = (response?: { entries: FileActivity[] }) => {
+    if (!response || !response.entries || !response.entries.length) {
+        return [];
+    }
+
+    const data = response.entries;
+    const parsedData = [];
+
+    data.map(item => {
+        if (!item.source) {
+            return item;
+        }
+
+        const source = { ...item.source };
+        // $FlowFixMe
+        const commentItem: Comment = source[FILE_ACTIVITY_TYPE_COMMENT];
```

Comment:
Should these be within the `case` statements?

Reply:
We get a `Unexpected lexical declaration in case block` eslint message

Reply:
Need to use block syntax for the case statements, which is generally a good idea anyway.

```
case:
    // whatever


case: {
    // whatever
}
```

Reply:
Did not know this was a thing, interesting, will do